src/client/Microsoft.Identity.Client/Cache/Items/MsalRefreshTokenCacheItem.cs

```diff
@@ -52,6 +52,8 @@ internal MsalRefreshTokenCacheItem()
         /// </summary>
         public string FamilyId { get; set; }
 
+        internal string UserAssertionHash { get; set; }
```

Comment:
This should be moved to the base class and removed from the child classes. This will allow us to simply code elsewhere. There are only 3 child classes so we should be ok

Reply:
Since `MsalIdTokenCacheItem` won't have `UserAssertion` I did not add it to the base class.